static/css/frame.scss

```diff
@@ -96,20 +108,26 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   margin: 0;
-  max-width: 600px;
 }
 
 .shot-title {
-  @include respond-to("small") {
-    font-size: 20px;
-    line-height: 20px;
-    padding-bottom: 4px;
+  @include respond-to("medium") {
+    max-width: 200px;
+  }
+  @include respond-to("large") {
+    font-size: 32px;
+    line-height: 39px;
+    max-width: 250px;
+    padding-bottom: 8px;
+  }
+  @media (min-width: 1260px) {
+    max-width: 500px;
```

Comment:
Added explicit check to increase title width when window size is more than 1260. Without this check header layout for title width 500px, wraps around when window size is 1000px